inspect.js

```diff
@@ -0,0 +1,526 @@
+
+    const canvas = renderer.domElement;
+    canvas.classList.add('side-content');
+    canvas.classList.add('view');
+    views.appendChild(canvas);
+  }
+  {
+    const {renderer, scene, camera} = _makeScene();
+
+    const modelMesh = await p.getModel();
+    scene.add(modelMesh);
+
+    const orbitControls = new OrbitControls(camera, renderer.domElement, document);
+    orbitControls.screenSpacePanning = true;
+    orbitControls.enableMiddleZoom = false;
+    orbitControls.update();
+
+    function animate(timestamp, frame) {
+      orbitControls.update();
+
+      renderer.render(scene, camera);
+    }
+    renderer.setAnimationLoop(animate);
+
+    const canvas = renderer.domElement;
+    canvas.classList.add('side-content');
+    canvas.classList.add('view');
+    views.appendChild(canvas);
+  }
+
+  // manifest
+  const manifestJson = p.getManifestJson();
+
+  const manifest = document.getElementById('manifest');
+  manifest.value = JSON.stringify(manifestJson, null, 2);
+
+  // files
+  const type = document.getElementById('type');
+  const name = document.getElementById('name');
+  const description = document.getElementById('description');
+  type.innerText = manifestJson.xr_type;
+  name.innerText = manifestJson.name || '';
+  description.innerText = manifestJson.description || '';
+
+  _renderFiles(p);
+};
+
+const _updateManifest = (p, manifestString) => {
+  const compileRawData = [{
+    url: '/manifest.json',
+    type: 'application/json',
+    data: manifestString,
+  }];
+
+  p.files.forEach(f => {
+    if (f.url.endsWith('/manifest.json')) return;
+
+    const filename = f.url.match(/([^/]+$)/)[1];
+    compileRawData.push({
+      url: `/${filename}`,
+      type: f.response.headers['content-type'],
+      data: f.response.body,
+    });
+  });
+
+  return new XRPackage(XRPackage.compileRaw(compileRawData));
+};
+
+const _bakePackage = async p => {
+  const b = new Blob([p.data], {
+    type: 'application/webbundle',
+  });
+  const srcWbn = URL.createObjectURL(b);
+  const iframe = document.createElement('iframe');
+  iframe.src = `bake.html?srcWbn=${srcWbn}`;
+  iframe.style.top = '-10000px';
+  iframe.style.left = '-10000px';
+  iframe.style.visibility = 'hidden';
+  document.body.appendChild(iframe);
+  const {screenshot, volume, aabb} = await new Promise((resolve, reject) => {
+    iframe.addEventListener('load', e => {
+      const _message = e => {
+        if (e.data && e.data.method === 'result') {
+          resolve(e.data.result);
+          window.removeEventListener('message', _message);
+        } else if (e.data && e.data.method === 'error') {
+          reject(e.data.error);
+          window.removeEventListener('message', _message);
+        }
+      };
+      window.addEventListener('message', _message);
+    });
+    iframe.addEventListener('error', err => {
+      reject(err);
+    });
+  });
+
+  // Baking logic from xrpk CLI: https://github.com/webaverse/xrpackage-cli/blob/master/commands/bake.js
```

Comment:
I don't know if I would reference files like this in comments. Chances are good that file won't exist in a month.